config/application.rb

```diff
@@ -129,6 +129,10 @@ class Application < Rails::Application
         origins IdentityCors.allowed_origins_static_sites
         resource '/api/analytics-events', headers: :any, methods: [:get]
         resource '/api/country-support', headers: :any, methods: [:get]
+        if IdentityConfig.store.in_person_enable_public_address_search
+          resource '/api/addresses', headers: :any, methods: [:post]
+          resource '/api/usps_locations', headers: :any, methods: [:post]
```

Comment:
if these are `POST` you're going to need to allow the `OPTIONS` verb for preflight requests, take a look at the OIDC token endpoint

Reply:
Yup, check out `routes.rb` where we handle options there as well, and you'll need the controller to respond to it.

if you're doing it on localhost it may not be a "real" CORS request and the browser may not try the preflight request, but deploying to a sandbox env should highlight that